modules/openapi-generator/src/main/resources/Java/libraries/okhttp-gson/pojo.mustache

```diff
@@ -153,7 +148,7 @@ public class {{classname}} {{#parent}}extends {{{.}}} {{/parent}}{{#vendorExtens
   public {{classname}} add{{nameInCamelCase}}Item({{{items.datatypeWithEnum}}} {{name}}Item) {
     {{#vendorExtensions.x-is-jackson-optional-nullable}}
     if (this.{{name}} == null || !this.{{name}}.isPresent()) {
-      this.{{name}} = JsonNullable.<{{{datatypeWithEnum}}}>of({{{defaultValue}}});
+      this.{{name}} = JsonNullable.<{{{datatypeWithEnum}}}>of({{{defaultValue}}}{{^defaultValue}}new ArrayList<>(){{/defaultValue}});
```

Comment:
Why only some libraries have handling for `uniqueItems`?

Reply:
good point. i think some libraries do not yet support `set` but I could be wrong. 

I'll later push another commit to support set regardless.

Reply:
done via https://github.com/OpenAPITools/openapi-generator/pull/14703/commits/252c7f8cbc3659345c5eb41e6f980cd2e998ff31